PWG/EMCAL/macros/AddTaskEmcalEmbeddingHelperData.C

```diff
@@ -0,0 +1,59 @@
+/**
+ * \file AddTaskEmcalEmbeddingHelperData.C
+ * \brief AddTask for AliAnalysisTaskEmcalEmbeddingHelperData
+ *
+ * The main implementation of the AddTask. 
+ * Same structure as AddTaskEmbeddingHelper
+ *
+ */
+AliAnalysisTaskEmcalEmbeddingHelperData *AddTaskEmcalEmbeddingHelperData()
```

Comment:
I'd suggest that you make this a static method in the class and then just make the .C macro a wrapper. Or we could make the static method in the base class into a template class, and then we could use it for both classes